waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates/GhRepo.hs

```diff
@@ -0,0 +1,31 @@
+module Wasp.Cli.Command.CreateNewProject.StarterTemplates.GhRepo
+  ( createProjectOnDiskFromGhRepoTemplate,
+  )
+where
+
+import Control.Monad.IO.Class (liftIO)
+import StrongPath (Abs, Dir, Dir', Path', Rel')
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.CreateNewProject.Common (throwProjectCreationError)
+import Wasp.Cli.Command.CreateNewProject.ProjectDescription (NewProjectAppName, NewProjectName)
+import Wasp.Cli.Command.CreateNewProject.StarterTemplates.Templating (replaceTemplatePlaceholdersInWaspFile)
+import Wasp.Cli.GithubRepo (GithubRepoRef, fetchFolderFromGithubRepoToDisk)
+import Wasp.Project (WaspProjectDir)
+
+createProjectOnDiskFromGhRepoTemplate ::
+  Path' Abs (Dir WaspProjectDir) ->
+  NewProjectName ->
+  NewProjectAppName ->
+  GithubRepoRef ->
+  Path' Rel' Dir' ->
+  Command ()
+createProjectOnDiskFromGhRepoTemplate absWaspProjectDir projectName appName ghRepoRef templatePathInRepo = do
+  fetchTheTemplateFromGhToDisk >>= either throwProjectCreationError pure
+  -- NOTE: Placeholders in main.wasp files of templates were probably not the best idea,
+  --   and we should likely drop the concept in the future, but we still do this replacement here
+  --   in order to keep it all working for old templates. Once we stop using any templates that
```

Comment:
We just need to replace placeholders with plain text and it will work. If placeholders are not found, the CLI won't complain.

Reply:
Hm you are right, I had another error at that time that confused me thinking it will be an issue, but it won't.
Would you suggest we do it then? The thing is, we need to put a specific Wasp version in them then. So for some older Wasp CLI's, it might be too new version. But well, that is the state of things really. Yeah, I think we should do it: in starters/repo on `main` branch I will set Wasp version to ^0.11 and remove placeholders for remove this templating code completely. Just let me know if you agree with this first.

Reply:
I think that's a good plan 👍 the `main` branch templates are locked forever to the `^0.11.0` version

Reply:
Ok, so what I did is following:
1. On `starters` repo I locked `main` branch (made it read-only), made new `dev` branch from it, made it default branch, and on that `dev` branch I removed templates.json, removed saas template, and pinned versions to 0.12. I also updated README in `main` to explain why it is locked, and I updated README in `dev` to be in sync with the new situation there.
2. Here in this repo I removed this code comment we commented about, and I also removed `version` from the placehodlers -> I left app name and title though, why not, if some template wants to use it it can.

So that is it -> old Wasp versions will work as they worked so far, while new ones (starting with 0.12) will be following the new approach.

At one point we might want to delete the `main` branch from `starters`, but it doesn't really matter even if we don't.

Reply:
Perfect 👍